ansible/roles/test/files/ptftests/IP_decap_test.py

```diff
@@ -178,18 +186,30 @@ def create_encap_packet(self, dst_ip, outer_pkt='ipv4', triple_encap=False):
         tc_in = tos_in = dscp_in << 2
         dscp_out = random.randint(0, 32)
         tc_out = tos_out = dscp_out << 2
-        if ("pipe" == self.test_params['dscp_mode']):
+        if "pipe" == self.test_params['dscp_mode']:
             exp_tc = exp_tos = tc_in
-        elif("uniform" == self.test_params['dscp_mode']):
+        elif "uniform" == self.test_params['dscp_mode']:
             exp_tc = exp_tos = tc_out
         else:
             print("ERROR: no dscp is configured")
             exit()
 
+        if outer_ttl is None:
+            outer_ttl = random.randint(2, 63)
```

Comment:
I'd prefer don't have randomness in the test. When we have randomness it'd be not easy to reproduce the test fails.
Let's use some predefined constant

Reply:
The ptf framework records the seed used for the test in log file, e.g.:
"13:20:13.364  root      : INFO    : Autogen random seed: 54164296"
If needed to reproduce/debug failed test it is possible to specify '--random-seed' in ptf command line the same that failed test has and have reproducible results.
Currently many tests use random input, maybe we can leave random as is for now?

Reply:
I have commited a fix which picks DSCP and TTL values from allowed range in a round robin way.

Reply:
@pavel-shirshov 
Could you please help review the changes and resolve the CR?